docker/Dockerfile-api

```diff
@@ -9,31 +9,18 @@ RUN dnf -y install \
     httpd \
     mod_auth_gssapi \
     mod_ssl \
-    python3-celery \
-    python3-flask \
-    python3-flask-login \
-    python3-flask-migrate \
-    python3-flask-sqlalchemy \
-    python3-GitPython \
-    python3-mod_wsgi \
-    python3-packaging \
     python3-pip \
-    python3-psycopg2 \
-    python3-requests \
-    python3-requests-kerberos \
-    python3-semver \
-    python3-setuptools \
+    python3-mod_wsgi \
```

Comment:
I can not avoid installing this package because I am experiencing problem (https://github.com/GrahamDumpleton/mod_wsgi/issues/76)

Reply:
Is it possible that we just need to install some C headers? I will look into it.

Reply:
Installing `httpd` and `httpd-devel` should fix it: https://github.com/GrahamDumpleton/mod_wsgi#system-requirements
As with any compiled python packages, we will also need `gcc` and `python-devel`